PowerShell.Common.props

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Product>PowerShell Core</Product>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
+
+    <FileVersion>6.0.0.0</FileVersion>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+    <NeutralLanguage>en-US</NeutralLanguage>
```

Comment:
I agree.
Done.

For clarify. I only found the VersionPrefix in Microsoft.NET.DefaultAssemblyInfo.targets where it define `Version` property.